pkg/reconciler/broker/broker.go

```diff
@@ -287,7 +288,7 @@ func (r *Reconciler) reconcileDecouplingTopicAndSubscription(ctx context.Context
 		topic, err = client.CreateTopicWithConfig(ctx, topicID, topicConfig)
 		if err != nil {
 			logger.Error("Failed to create Pub/Sub topic", zap.Error(err))
-			b.Status.MarkTopicFailed("CreationFailed", "Topic creation failed: %w", err)
+			b.Status.MarkTopicFailed("TopicCreationFailed", "Topic creation failed: %w", err)
```

Comment:
Surprising to me that we don't need to change unit test accordingly?

Reply:
I don't think we test this case currently. Maybe it's not easy to inject an error in pstest?